library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -107,7 +108,7 @@ protected void starting(org.junit.runner.Description description)
       fileSplitterInput.setScanner(scanner);
       fileSplitterInput.getScanner().setScanIntervalMillis(500);
       fileSplitterInput.getScanner().setFilePatternRegularExp(".*[.]txt");
-      fileSplitterInput.getScanner().setFiles(dataDirectory);
+      fileSplitterInput.getScanner().setFiles(new File(dataDirectory).getAbsolutePath());
```

Comment:
@DT-Priyanka 
The user can add relative or absolute path. We shouldn't be imposing adding absolute path on the user.